torchvision/prototype/transforms/functional/utils.py

```diff
@@ -0,0 +1,93 @@
+import functools
+import inspect
+from typing import Any, Type, Optional, Callable
+
+import PIL.Image
+import torch
+import torch.overrides
+from torchvision.prototype import features
+from torchvision.prototype.utils._internal import sequence_to_str
+
+
+def is_supported(obj: Any, *types: Type) -> bool:
+    return (obj if isinstance(obj, type) else type(obj)) in types
```

Comment:
It will be used by some transforms later on. See for example https://github.com/pytorch/vision/blob/16a4473519b2105154b33f34083c62de9c00396c/torchvision/prototype/transforms/_misc.py#L29. I would keep it as function or otherwise I'll make it a function again for the next PR.